include/circt/Dialect/GAA/GAA.td

```diff
@@ -0,0 +1,67 @@
+//===- GAA.td - GAA dialect definition ---------------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This is the top level file for the Guard-Atomic-Action dialect.
+//
+//===----------------------------------------------------------------------===//
+#ifndef GAA_TD
+#define GAA_TD
+
+include "mlir/IR/OpBase.td"
+include "mlir/IR/OpAsmInterface.td"
+include "mlir/IR/SymbolInterfaces.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/IR/RegionKindInterface.td"
+
+// depends on FIRRTL types
+include "circt/Dialect/FIRRTL/FIRRTLDialect.td"
+include "circt/Dialect/FIRRTL/FIRRTLTypes.td"
+
+def GAADialect : Dialect {
+  let name = "gaa";
+
+  let summary = "Types and operations for gaa dialect";
+  let description = [{
+    This dialect defines the `gaa` dialect, which is used to lower from
+    Chisel GAA code to Verilog.  For more information, see the
+    [Chisel-GAA GitHub page](https://github.com/sequencer/gaa).
```

Comment:
Link isn't working?

Reply:
Yes, I'm re-prototyping this, I'll update this today.